packages/replay/metrics/src/results/metrics-stats.ts

```diff
@@ -11,6 +11,10 @@ export class MetricsStats {
   static cpu: NumberProvider = metrics => metrics.cpu.average;
   static memoryMean: NumberProvider = metrics => ss.mean(Array.from(metrics.memory.snapshots.values()));
   static memoryMax: NumberProvider = metrics => ss.max(Array.from(metrics.memory.snapshots.values()));
+  static netTx: NumberProvider = metrics => ss.sum(metrics.network.events.map(e => e.requestSize || 0));
```

Comment:
Sorry if the question is stupid, but are `tx` and `rx` standard terms? TBH I didn't understand them, if they are not standard terms, maybe we could use something more explicit?

Reply:
well, [they are in relation to networking](https://www.google.com/search?q=network+tx+and+rx) but I'm open to suggestions